build/periodic.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
+
+# Validate the pipeline is up to date and that no failed prow jobs exist
+
+COMPONENT_ORG=stolostron
+#COMPONENT_ORG=open-cluster-management
+CHECK_RELEASES="2.3 2.4 2.5"
+
+REPOS="$COMPONENT_ORG/cert-policy-controller
+$COMPONENT_ORG/config-policy-controller
+$COMPONENT_ORG/governance-policy-propagator
+$COMPONENT_ORG/governance-policy-spec-sync
+$COMPONENT_ORG/governance-policy-status-sync
+$COMPONENT_ORG/governance-policy-template-sync
+$COMPONENT_ORG/iam-policy-controller"
+
+# Clone the repositories needed for this script to work
+cloneRepos() {
+	#git clone https://github.com/$COMPONENT_ORG/policy-grc-squad.git
```

Comment:
Should we restore this line?

Reply:
I restored it so the repo list is calculated now instead of hard coded